Sources/WhisperKit/Core/WhisperKit.swift

```diff
@@ -371,42 +359,101 @@ public class WhisperKit: Transcriber {
 
     public func clearState() {
         audioProcessor.stopRecording()
-        currentTimings = nil
+        currentTimings = TranscriptionTimings()
     }
 
     deinit {
-        clearState()
+        audioProcessor.stopRecording()
     }
 
     /// Pass in your own logging callback here
     public func loggingCallback(_ callback: Logging.LoggingCallback?) {
         Logging.shared.loggingCallback = callback
     }
 
-    // MARK: - Transcribe audio file
+    // MARK: - Transcribe multiple audio files
+
+    public func transcribe(
+        audioPaths: [String],
+        decodeOptions: DecodingOptions? = nil,
+        callback: TranscriptionCallback = nil
+    ) async -> [Result<[TranscriptionResult], Swift.Error>] {
```

Comment:
I support moving to `Result` types, but I think it should be gradual so as not to be jarring compared to the other single audio interfaces. Can you add support for both optional throwing and non-optional result formats of this interface, similar to the other convenience methods?

Reply:
added additional method without `Result`